src/GraphCool.php

```diff
@@ -156,7 +156,7 @@ protected function sendResponse(array $response): void
     {
         header('Content-Type: application/json');
         $env = Env::get('APP_ENV');
-        if ($env === 'local' || $env === 'test') {
+        if ($env === 'local' || $env === 'test' || $env === 'staging') {
```

Comment:
würde das mal sinn machen in ne funktion auszulagern? also `!isProduction()`, so muss man dann später nur noch an einer stelle den code anpassen

Reply:
möglich, aber eventuell ändert sich das ja wieder, dass es für staging nicht mehr erwünscht ist. oder das ich es mal - vorübergehend in production auch live haben will, oder dass es für test zu entfernen wäre...

Reply:
das könnte man ja auch hinter funktionen besser verstecken, ala `appendDebugTime(): bool` usw. nur eine idee man diese ifs else ors benennt